corehq/util/celery_utils.py

```diff
@@ -186,3 +190,45 @@ def get_running_workers(timeout=10):
         worker_names.extend(worker_info.keys())
 
     return worker_names
+
+
+class LoadBasedAutoscaler(Autoscaler):
+    def _maybe_scale(self, req=None):
+        procs = self.processes
+        cur = min(self.qty, self.max_concurrency)
+
+        available_cpus = multiprocessing.cpu_count()
+        load_avgs = os.getloadavg()
+        max_avg = max(load_avgs[0], load_avgs[1])
+        normalized_load = max_avg / available_cpus
+
+        if cur > procs and normalized_load < 0.90:
```

Comment:
if the load is ~1 then the machine is totally saturated right? I was thinking that may make it hard to log in and debug issues. 0.90 seemed like pretty good utilization while still allowing for us to log in